### ISSUE TITLE
Seacan hardcodes running directory to `/tmp/`

### ISSUE THREAD

**Issue by xsawyerx** [xsawyerx/pakket#79]
One bug I introduced with changes to the Seacan script is a bug that it builds into `/tmp/` and hard-codes the path.

**Comment by gonzus** [comment]
Can you show how to reproduce the bug and expected vs actual results? Even better, a test that fails? :-)

**Comment by gonzus** [comment]
I guess I am not understanding the issue fully... The line that builds the final tar is this:

```
system( 'tar', '--create', '--gzip', '--exclude-vcs', "--directory=$dest_dir", '--file=' . path('pakket.tar.gz')->absolute, '.', );
```

Although $dest_dir is a directory in /tmp, the --directory changes the cwd there first, so the tar that is created should be relative to '.'. And this is happening (I tested it).

So, what's the problem again?

**Comment by xsawyerx** [comment]
Check the bin created in the Seacan dir when you open it: `bin/pakket`. It hard-codes the path to the Pakket directory, the Perl bin directory, and the local lib directory.

**Comment by xsawyerx** [comment]
Resolved by fe3ecc6. Finally.
